src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -131,7 +131,7 @@ private void greedyAllocate(BitSet remainingPeople, Set<Household.HouseholdType>
                 }
                 remainingPeople.clear(i);
                 aPopulation[i].setHome(population[h]);
-                population[h].addPerson(aPopulation[i]);
+                population[h].addInhabitant(aPopulation[i]);
```

Comment:
Is setHome() being called twice for everyone?  (I guess this isn't a big problem.)

Reply:
Looks like it, It makes most sense to handle this inside the Household class so I'll delete l133